### PR TITLE
Adding OSM as an alpha state adapter

### LINKED ISSUE #282: Add entry for Open Service Mesh
**Current Behavior**
Meshery has an alpha adapter for Open Service Mesh in the `meshery-osm` repo.

**Desired Behavior**
This alpha state adapter should be represented on meshery.io

OSM logo - 
[osm.svg.zip](https://github.com/layer5io/meshery.io/files/5039020/osm.svg.zip)

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

This PR fixes #282 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by manishdangi98** [comment]
@leecalcote We need to fix a few things in mobile view

- hem burger icon in the navbar needs to be shifted towards the right

- footer needs to be center-aligned

Screenshot 2020-08-09 at 3 45 20 PM

- And "Overview" can be center-aligned  
WhatsApp Image 2020-08-09 at 3 43 31 PM

File: index.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 title: Meshery | The Service Mesh Management Plane
-description: Quickly compare and learn service meshes like Istio, Linkerd, Envoy, Octarine, App Mesh, Consul, Kuma, Maesh, Tanzu.
-Lifecycle, performance and configuration management across different service meshes. Patterns and best practices of service mesh operation.
+description: Quickly compare and learn service meshes like Istio, Linkerd, Envoy, Octarine, App Mesh, Consul, Kuma, Maesh, Tanzu. Lifecycle, performance and configuration management across different service meshes. Patterns and best practices of service mesh operation.
```
Comment (PluckyPrecious) [inline index.html:3]: What do you think about adding “osm” to those listed in the description?

Comment (leecalcote) [reply index.html:3]: Great suggestion, @PluckyPrecious. Yes, totally.

**Comment by leecalcote** [comment]
https://github.com/layer5io/meshery.io/pull/285#issuecomment-671045934

Good eye, @manishdangi98. Here's a new issue for this - https://github.com/layer5io/meshery.io/issues/288. You might consider tackling this one. 👍

File: index.html (hunk 1 above)
Comment (leecalcote) [reply index.html]: Added OSM to the page description.

**Comment by leecalcote** [comment]
Thanks for these reviews, @PluckyPrecious @Ruth-ikegah @manishdangi98

**Pull request merged**